docs/docs/about/why-cli.mdx

```diff
@@ -2,8 +2,47 @@
+- Make [CLI for Microsoft 365 your own](../user-guide/configuring-cli). CLI for Microsoft 365 offers a variety of settings that will allow you to customize it for your specific use case or needs. If going over the settings is challenging, then you will love the [m365 setup](../cmd/setup) command that will help you customize CLI for common use cases like setting it to be tailored to be run in a script.
+- AI, Of course, we have an AI! CLI for Microsoft 365 comes along with an [AI assistant chili 🌶️](../user-guide/chili) that will help you find the right commands for your tasks.
+- You may use it in CI/CD pipelines. It integrates easily with Azure DevOps and natively with GitHub workflows. CLI has dedicated [GitHub actions](../user-guide/github-actions) which you may use directly in GitHub workflow and it allows you to scaffold a pipeline for [Azure DevOps](../cmd/spfx/project/project-azuredevops-pipeline-add) and [GitHub workflows](../cmd/spfx/project/project-github-workflow-add).
+- How many CLI's allows you to run it as your API in an app? That is what I though. [CLI for Microsoft 365 may be used as an API](../user-guide/use-cli-api) in your Node.js app.
+- You may filter and sort the output of every command. [Filtering and sorting](../user-guide/filter-cli-data.mdx) is a powerful feature that allows you to get only the data you need and in the order you want.
+
+Interested in more? Start with our [user guide](../user-guide/using-cli.mdx) and learn how to use the CLI for Microsoft 365.
+
+# Why not another approach or a different CLI?
+
+### Why not just do things manually?
+
+Using CLI for Microsoft 365 is not about replacing the UI, it's about making your life easier. If you need to do a repetitive task, like creating a new SharePoint list every time a new project is started, you can create a script and automate this task with CLI for Microsoft 365. This way you can save time and avoid human errors.
+
+### Why not just write your own CLI or some custom console app tailored to your needs?
+
+You could, but why reinvent the wheel? CLI for Microsoft 365 is a powerful tool that is being actively developed and maintained by a team of experts. It's open-source and free to use. It's also cross-platform and works on any device. It's a great tool that can help you manage your Microsoft 365 tenant more efficiently.
+
+### Why not use SharePoint Online Management Shell?
+
+SharePoint Online Management Shell is a great tool, but it's limited to SharePoint Online. CLI for Microsoft 365 offers a variety of commands from many different areas of Microsoft 365 like SharePoint Online, Teams, Viva engage, Entra, Power Apps, Power Automate, Power Platform, OneDrive, Planner, OneNote, Outlook, SharePoint Embedded and many more. It's a more comprehensive tool that can help you manage your Microsoft 365 tenant more efficiently.
```

Comment:
Also, let's emphasize that it works only in PowerShell. Does it still work only on Windows?

Reply:
AFAIK, this module is still not supported with PS Core, right? So I guess it's only Windows yeah.
Also, another downside that you could mention is that the module only works for SharePoint administration tasks. So you can only use it to run admin-related commands. Doing stuff on a regular SP site is not supported.

Reply:
yep. That is correct it works only on Windows PS